CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ### New Features
 
+-
```

Comment:
```suggestion

```

Reply:
I usually leave this as a placeholder, I find having a non-empty section makes it a bit more readable :)